runtime/doc/options.txt

```diff
@@ -8407,6 +8422,64 @@ A jump table for the options with a short description can be found at |Q_op|.
 	Maximum number of tab pages to be opened by the |-p| command line
 	argument or the ":tab all" command. |tabpage|
 
+				*'tabsidebar'* *'tsb'* *g:actual_curtabpage*
+'tabsidebar' 'tsb'			string	(default empty)
+			global
+			{not in Vi}
+	When non empty, this option determines the content of the tabsidebar.
+	The option consists of printf style '%' items interspersed with
+	normal text, similar to the |'statusline'| or |'tabline'|.
+
+	You can use |g:actual_curtabpage| within a function assigned to
```

Comment:
The name `g:actual_curtabpage` looks a bit off. Actual and current? 

And shouldn't it be not global but, `v:tabpagenr` available when tabsidbar option is set up? Or even `v:lnum` which is available for `guitablabel`?

Reply:
This follows like `g:actual_curbuf` or `g:actual_curwin`.

Reply:
I see